websites/R/RealLifeGTAV/presence.ts

```diff
@@ -11,6 +11,7 @@ presence.on("UpdateData", async function () {
     },
     set_showUsername = await presence.getSetting("showUsername"),
     set_timeElapsed = await presence.getSetting("timeElapsed"),
+    set_showButtons = await presence.getSetting("showButtons"),
     urlpath = window.location.pathname.split("/");
   function ifSettingEnabled(setting, string) {
```

Comment:
What? Just use a ternary operator in place of this wherever you need to check a setting is toggled... 
![X48vo3BzLw](https://user-images.githubusercontent.com/34070194/110235593-c8402b00-7f28-11eb-8f81-81fde4cb388e.png) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator